torchvision/models/detection/backbone_utils.py

```diff
@@ -111,7 +111,8 @@ def _resnet_fpn_extractor(
 ) -> BackboneWithFPN:
 
     # select layers that wont be frozen
-    assert 0 <= trainable_layers <= 5
+    if trainable_layers not in range(0, 6):
```

Comment:
range() is a bit slow and this of course takes O(n) time to check. I suggest a simpler logic 
```suggestion
    if trainable_layers < 0 or trainable_layers > 5:
```

Reply:
okay! Understood